WLogger/WeekLogs/LogTableViewCell/LogViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  LogViewModel.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/10/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+
+public struct LogViewModel {
```

Comment:
Here I would recommend to use computed properties

```swift
public struct LogViewModel {

    private let _log: Log

    public var projectName: String {
        return log.projectName
    }

    public var activityName: String {
        return log.activityName
    }

    public var comments: String {
        return log.comments
    }

    public let duration: String

    public init(log: Log) {
        _log = log
        duratio = secondsStringFormatted(log.duration)
    }

}
```

And ideally if you want to respect the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter) you should add the following methods to Log

```swift
extension Log {

    public var projectName: String {
        return project.name
    }

    public var activityName: String {
        return activity.name
    }

}
```